Enhancements_Fast_api/main.py

```diff
@@ -0,0 +1,231 @@
+from fastapi import FastAPI, status, HTTPException, Depends
```

Comment:
is this file required, I think we already have the routes.py file.

Reply:
Yes I'll remove it.